specs/protocol/jovian/exec-engine.md

```diff
@@ -82,20 +83,30 @@ contract values to the block builder via `PayloadAttributesV3` parameters.
 
 ## DA Footprint Block Limit
 
-From Jovian, `gasUsed` is equal to a block's "DA footprint" if the footprint exceeds
-the total gas used by transaction, and equal to the sum of the gas used by each transaction (excluding deposits)
-otherwise. As a result, blocks with high DA usage may cause the base fee to increase in subsequent blocks.
-
-A block's DA footprint is calculated by scaling the cumulative DA footprint of its transactions
-(as calculated by the [Fjord LZ Estimation](../fjord/exec-engine.md#fjord-l1-cost-fee-changes-fastlz-estimator) by
-a configurable scalar value, the `daFootprintGasScalar`:
+Let a block's `scaledDAFootprint` be defined as follows:
 
 ```python
-da_usage_estimate = max(minTransactionSize, intercept + fastlzCoef*fastlzSize / 1e6)
-da_footprint = da_usage_estimate * da_footprint_gas_scalar
+def scaledDAFootprint(block)
```

Comment:
Might be best to go back and align this with the terminology in the design doc. We already have the "size estimate" in the Fjord specs.

Reply:
done [0cb6085](https://github.com/ethereum-optimism/specs/pull/776/commits/0cb60859b89f1f4e80171ee33e8a4339d792fb47)